Detector/DetSegmentation/include/DetSegmentation/FCCSWGridModuleThetaMerged.h

```diff
@@ -63,7 +63,7 @@ class FCCSWGridModuleThetaMerged : public GridTheta {
    *   return The number of merged cells in theta
    */
   inline int mergedThetaCells(const int layer) const {
-    if (layer<m_mergedCellsTheta.size())
+    if (layer < (int) m_mergedCellsTheta.size())
```

Comment:
Are you sure that `m_mergedCellsTheta.size()` will not be larger than int numeric limit?

Reply:
yes, cells are grouped in small numbers (typically between 1 and 4)

Reply:
OK, can you add this comment to the code?

Reply:
sorry answer was not correct - you asked about size() of the vector, not about the content of the vector. The size of the vector is equal to the number of layers of the calo, which is 12 in the baseline design and anyway O(10)

Reply:
now that I clarified that the size of the vector is equal to nLayers (as written also when the vector is declared, and do you think I should really add this comment?

Reply:
Yes, to have the justification for the casting to `int` directly in the code.

Reply:
Nice